docs/rest_api.md

```diff
@@ -2,3 +2,266 @@
+
+Create a trigger with name `events` with a default parameter `type` with value `webhook` set.
+```bash
+curl -u $AUTH -d '{"name":"events","parameters":[{"key":"type","value":"webhook"}]}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/events?overwrite=true 
+```
+
+Now whenever you have an event that needs to fire this trigger it just takes a HTTP request with a method `POST` using the OpenWhisk Authorization key.
+
+To fire the trigger `events` with a parameter `temperature` send the following HTTP request.
+
+```bash
+curl -u $AUTH -d '{"temperature":60}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/events
+```
+
+### Triggers with Feed Actions
+
+There are special triggers that can be created using a feed actions, the feed action it's an action that helps with the configuration of a feed provider that will be in charge of firing the trigger whenever there is an event for the trigger. Learn more how these feed providers in the [feeds.md] documentation.
+
+Some of the available triggers that leverage a feed action are periodic/alarms, slack, github, cloudant/couchdb, and messageHub/kafka. You also can create your own feed action and feed provider.
+
+Let's create a trigger with name `periodic` to be fire at a specified frequency, every 2 hours (i.e. 02:00:00, 04:00:00, ...).
+
+Using the CLI this will be done with one command
+```bash
+wsk trigger create periodic --feed /whisk.system/alarms/alarm \
+  --param cron "0 */2 * * *" -v
+```
+As you will see because we are using the `-v` flag is that two HTTP requests are sent, one is to create a trigger `periodic` and the other is to invoke a feed action `/whisk.system/alarms/alarm` with the parameters to configure the feed provider to fire the trigger every 2 hours.
+
+To do the same with the REST API, lets create the trigger first
+```bash
+curl -u $AUTH -d '{"name":"periodic","annotations":[{"key":"feed","value":"/whisk.system/alarms/alarm"}]}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/periodic?overwrite=true 
+```
+
+As you can see the annotation `feed` is stored in the trigger which later we are going to use to know which feed action to use when deleting the trigger. 
+
+Now that the trigger is created, lets invoke the feed action
+```bash
+curl -u $AUTH -d "{\"authKey\":\"$AUTH\",\"cron\":\"0 */2 * * *\",\"lifecycleEvent\":\"CREATE\",\"triggerName\":\"/_/periodic\"}" -X POST -H "Content-Type: application/json" "https://openwhisk.ng.bluemix.net/api/v1/namespaces/whisk.system/actions/alarms/alarm?blocking=true&result=false"
+```
+
+Deleting the trigger, it's a similar process as creating the trigger, this time deleting the trigger and also using the feed action to configure the feed provider to delete the handler for the trigger.
```

Comment:
Deleting the trigger, it's a similar process as ->Deleting the trigger is similar to